src/data/taxonomy-redirects.json

```diff
@@ -2501,5 +2519,77 @@
     "paths": [
       "/docs/using-new-relic/cross-product-functions/install-configure"
     ]
-  }    
+  },
+  {
+    "url": "/docs/apm/new-relic-apm/getting-started/introduction-apm",
+    "paths": [
+      "/docs/apm"
+    ]
+  },
+  {
+    "url": "/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring",
+    "paths": [
+      "/docs/synthetics"
+    ]
+  },
+  {
+    "url": "/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring",
+    "paths": [
+      "/docs/infrastructure"
+    ]
+  },
+  {
+    "url": "/docs/logs/get-started/get-started-log-management",
+    "paths": [
+      "/docs/logs"
```

Comment:
@paperclypse, this 404s, too, so I'm guessing that my local system can't resolve any of these new entries.